src/styles/blocks/stopwatch.scss

```diff
@@ -0,0 +1,76 @@
+.stopwatch {
+  &__container {
+    position: absolute;
+    top: 50%;
+    left: 50%;
+    transform: translate(-50%, -50%);
+
+    background: rgb(2,0,30);
+    height: 500px;
+    width: 500px;
+    border-radius: 50%;
+  }
+
+  &__center-circle {
+    position: absolute;
+    top: 50%;
+    left: 50%;
+    transform: translate(-50%, -50%);
+    @include circle ($stopwatch-circle-diametr);
+
+    background-color: $color-grey;
+    box-shadow: 0 0 10px black;
+  }
+
+  &__minutes-hand {
+    position: absolute;
+    left: 48.5%;
```

Comment:
[STYLES] - If you need to adjust positioning with 1px or fractional numbers like 0.5, you are doing something wrong, change the alignment technique you use